pom.xml

```diff
@@ -559,7 +539,7 @@
             <plugin>
                 <groupId>com.coveo</groupId>
                 <artifactId>fmt-maven-plugin</artifactId>
-                <version>2.0.0</version>
+                <version>2.3.0</version>
```

Comment:
❓ Any formatting changes that people need to be aware of?

Reply:
Only that they added a skip flag. 
There should *never* be any formatting changes **ever** with the google formatter. The style is set in stone and should never change - unless we change formatters.

Reply:
Not that I had run into yet